src/utils/themeMap.tsx

```diff
@@ -450,7 +450,7 @@ export const typesMap: TTypesMap = {
     ),
     easing: ({ value, onChange, ...props }) => <FieldEditor {...props} value={value} onChange={onChange} />,
     fontFamily: ({ value, onChange, ...props }) => <FieldEditor {...props} value={value} onChange={onChange} />,
-    fontWeight: ({ value, onChange, ...props }) => <FieldEditor {...props} value={value} onChange={onChange} />,
+    fontWeight: ({ value, onChange, ...props }) => <FontWeightEditor {...props} value={value} onChange={onChange} />,
```

Comment:
can we simplify this to ???
```suggestion
    fontWeight: FontWeightEditor,
```